.github/pull_request_template.md

```diff
@@ -1,3 +1,4 @@
+Jira Ticket: [PXP-xxxx](https://ctds-planx.atlassian.net/browse/PXP-xxxx)
```

Comment:
PRs get automatically linked to the JIRA ticket when we add "PXP-xxx" in the PR title, this feels a bit like a step backwards to me

we can add "use jira number" as a reminder in the list below... assuming people read it :/

Reply:
Right, I couldn't figure out how to make a default PR title (I don't think it's possible?). I do think it would be helpful to have a link to the jira in the PR, so you can go to the ticket from the PR to get more context on the ticket when reviewing.

Reply:
I could add a lil checkbox below this saying "- [ ] I put the Jira number in the title" or something!

Reply:
Oh, it is possible to change the title -- I'll look into this: https://help.github.com/en/github/building-a-strong-community/about-issue-and-pull-request-templates#issue-templates

Reply:
it doesn't look like you can do it for PRs 😢 
the automatic linking with the title creates a link from jira to github, but not the opposite, so we can do what you said (both link in the description and in the title) and get feedback during retro to see if it's too much

Reply:
You're right about the PR title 😿. Yeah I'll put a hopefully just-annoying-enough checkbox under the link.